### DISCUSSION TITLE
Failed to update dependencies: Request entity too large

### DISCUSSION THREAD

**Discussion by dtim-gmu** [deta/cloud-docs#261] [Help] [answered]
I'm trying to deploy a very small application requiring only 9 dependencies. Apparently, the size of these dependencies exceed the limit of 250MB through which I'm unable to deploy the application. Step `deta deploy` failes with error `failed to update dependencies: Request entity too large`. I've checked the size of each package and this is nowhere close to 250MB in total. Any suggestions on how to solve this issue? 

EDIT: After further investigation of each package, it turns out the `osmnx` package causes trouble. Locally the size of this package is 0.5MB so I'm trying to figure out what happens during deployment.

**Comment by lemonyte** [comment, marked as answer]
[`osmnx`](https://pypi.org/project/osmnx/) depends on [`Fiona`](https://pypi.org/project/Fiona/), which requires GDAL to be installed, which cannot be done in a micro. This may be the reason your dependencies fail to install. Image below is me trying to install `osmnx` on Windows in a fresh virtual environment with no GDAL available.
![image](https://user-images.githubusercontent.com/49930425/184516666-ae628366-fd0b-4787-bcf2-c15ba70d99c4.png)
